### PR TITLE
readme badges

### PR DESCRIPTION
**Pull request by Carreau**
capture d ecran 2015-01-08 a 18 47 32

Just mostly because otherwise it's hard to find the coverage link. 
So I also added a few other.

We can also be fancy with

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,16 @@
+.. image:: https://img.shields.io/coveralls/ipython/ipython.svg 
+    :target: https://coveralls.io/r/ipython/ipython?branch=master
+
+.. image:: https://img.shields.io/pypi/dm/IPython.svg           
+    :target: https://pypi.python.org/pypi/ipython
+
+.. image:: https://img.shields.io/pypi/v/IPython.svg            
+    :target: https://pypi.python.org/pypi/ipython
```
Comment (minrk) [inline README.rst]: I see there are two copies of pypi

Comment (minrk) [inline README.rst]: Ignore me, I see v vs dm

Comment (Carreau) [inline README.rst]: For future reader, `v` is version and `dm` download per month.

**Comment by damianavila** [comment]
Sounds like a good idea...

**Comment by ellisonbg** [comment]
Travis is failing on this one...

**Comment by takluyver** [comment]
It seems unlikely that the test failure could be related to this change. I've prodded the relevant job to restart.

**Comment by takluyver** [comment]
Still failing. Looks like it's related to #7132, which was recently merged. I think this should be fine to merge, but can you (@Carreau) take a quick look?

**Comment by Carreau** [comment]
Rebased, also included #7407 from @minrk that needed a rebase too.

**Comment by ellisonbg** [comment]
Is this ready then?

**Comment by rgbkrk** [comment]
Looking good to me.

**Comment by rgbkrk** [comment]
The Travis updates I already merged with #7407, so this is just the README updates really now.

**Pull request merged**